src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/AotTests.cs

```diff
@@ -88,7 +88,6 @@ public void BuildBasicApplicationReleaseProfiledAotWithoutDefaultProfile ()
 
 		[Test]
 		[TestCaseSource (nameof (AotChecks))]
-		[Category ("SmokeTests")]
 		public void BuildAotApplicationAndÜmläüts (string supportedAbis, bool enableLLVM, bool expectedResult)
```

Comment:
I do worry about not running at least some of this (and the Debugger test) during a PR build. Things like mono bumps can break these and unless we are careful we might miss something. 

One idea might be to keep the Category, but re work the TestCaseSource to be a method which returns the list of tests to run (we have done this before https://github.com/xamarin/xamarin-android/blob/6ef8ba47f2912cb7fe310924a8b85515a41d43a1/tests/MSBuildDeviceIntegration/Tests/DeploymentTest.cs#L206). The then might be able to query if we are in "SmokeTest" mode and only run a limited set of these (even just 1) to make sure we don't break anything.

Reply:
My intent here was to reduce the default checks that run, and not to disable the ability to run all of our tests on any given PR. This does add a manual step that may make the authors work more burdensome in some cases, and as a result I am looking at having variability in the test stages that will run by default based on the code that is changing.

Reply:
Mono bumps should now result in every test job running automatically on each commit to the PR with the .external file change.